lib/http/client.rb

```diff
@@ -98,6 +103,11 @@ def verify_connection!(uri)
       # lazily load the body as long as possible, and this mimics prior functionality.
       elsif @connection && (!@connection.keep_alive? || @connection.expired?)
         close
+
```

Comment:
I find it makes it harder to read personally, but I don't really care either way.